tools/merlin/merlin.xml~

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="merlin" name="Tool for Linkage Analysis" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package">merlin</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    merlin -p $pname -m $mname -d $dname --quiet --error --best --npl > $merlinoutput
+    ]]></command>
+    <inputs>
+	<param name="pname" type="data" format="txt" help="Set of data values" />
+	<param name="mname" type="data" format="txt" help="Set of map values" />
+	<param name="dname" type="data" format="txt" help="Set of pedigree values" />
+    </inputs>
+    <outputs>
+	<data name="errors" format="txt" from_work_dir="merlin.err" />
+	<data name="flow" format="txt" from_work_dir="merlin.flow" />
+	<data name="chromosome" format="txt" from_work_dir="merlin.chr" />
+	<data name="merlinoutput" format="lod" />
+    </outputs>
+    <tests>
+	<test>
+	    <param name="pname" value="parametric.dat"/>
+	    <param name="mname" value="parametric.map"/>
+	    <param name="dname" value="parametric.ped"/>
+	    <output name="merlinoutput" file="parametric.lod"/>
+	</test>
+    </tests>
+    <help>
+	<![CDATA[
+		 MERLIN 1.1.2 - (c) 2000-2007 Goncalo Abecasis
+
+References for this version of Merlin:
+
+Abecasis et al (2002) Nat Gen 30:97-101        [original citation]
+Fingerlin et al (2004) AJHG 74:432-43          [case selection for association studies]
+Abecasis and Wigginton (2005) AJHG 77:754-67   [ld modeling, parametric analyses]
+Fingerlin et al (2006) Gen Epidemiol 30:384-96 [sex-specific maps]
+Chen and Abecasis (2007) AJHG 81:913-26        [qtl association analysis, qtl simulation]
+
+
+The following parameters are in effect:
+Data File :      merlin.dat (-dname)
+Pedigree File :      merlin.ped (-pname)
+Missing Value Code :         -99.999 (-xname)
+Map File :      merlin.map (-mname)
+Allele Frequencies : ALL INDIVIDUALS (-f[a|e|f|m|file])
+Random Seed :          123456 (-r9999)
+
+Data Analysis Options
+General : --error, --information, --likelihood, --model [param.tbl]
+IBD States : --ibd, --kinship, --matrices, --extended, --select
+NPL Linkage : --npl, --pairs, --qtl, --deviates, --exp
+VC Linkage : --vc, --useCovariates, --ascertainment, --unlinked [0.00]
+Association : --infer, --assoc, --fastAssoc, --filter, --custom [cov.tbl]
+Haplotyping : --best, --sample, --all, --founders, --horizontal
+Recombination : --zero, --one, --two, --three, --singlepoint
+Positions : --steps, --maxStep, --minStep, --grid, --start, --stop
+LD Clusters : --clusters [], --distance, --rsq, --cfreq
+Limits : --bits [24], --megabytes, --minutes
+Performance : --trim, --noCoupleBits, --swap, --smallSwap
+Output : --quiet, --markerNames, --frequencies, --perFamily, --pdf,
+--tabulate, --prefix [merlin]
+Simulation : --simulate, --reruns, --save, --trait []
+
+
+FATAL ERROR - The datafile merlin.dat cannot be opened
+ ]]>
+
```

Comment:
The help text needs to explain what the tool does to users without bombarding them with technical information.

Could you put together a small paragraph on what Merlin is, based on the intro text [here](http://csg.sph.umich.edu/abecasis/merlin/index.html) and the tool abstract [here](https://www.ncbi.nlm.nih.gov/pubmed/11731797)